plasma/plasma-mvp/morevp-alice-bob.md

```diff
@@ -0,0 +1,103 @@
+MoreVP Alice-Bob Scenarios
+---
+
+Note that these scenarios work with N participants, not just limited to 1 input 1 output. 
+
+Also, note that in all of these scenarios the exits are given the priority of the youngest input to the transaction.
+
```

Comment:
Maybe it's worth to make a separate short section about the exact behavior of the "piggyback state machine", if such a thing can be abstracted out?

Reply:
@kfichter bump this question/suggestion.

I keep bumping into not knowing exactly, how does piggybacking drive what get's to exit and what doesn't. 

We reached an interim understanding where the rule was that if any output is piggybacked, then only outputs get to exit, otherwise inputs get to exit, but after revisiting the MoreVP paper and these examples I start to doubt that this is the intention and that it is more convoluted with canonical-lity.

Such piggybacking-state-machine or playbook would be super useful, as a generalization of the things listed in [other question here](https://github.com/omisego/research/pull/42/files#r200910466)

Reply:
See my comment + image on the other question. Piggybacking has no impact on the validity of the transaction (whether inputs or outputs get to exit). Piggybacking is just a mechanism to attach a bond to each input or output that *might* get to exit. Because it's not known in advance whether the transaction is valid or not, owners of both inputs and outputs have to piggyback preemptively.